languages/javascript/concepts/equality/introduction.md

```diff
@@ -0,0 +1,66 @@
+**Equality** in the javascript can be referred in two ways which is `==` or `===`
+It is usually a misconception that people consider `==` which is usually check for value equality and `===` which usually check for the equality and type of the comparison which is not true.
```

Comment:
This starts off by saying that there is a misconception but does not say what that misconception is? This is also a run on sentence that would be easier to understand if broken down into smaller sentences.